chainer/gradient_check.py

```diff
@@ -211,7 +213,7 @@ def check_backward(func, x_data, y_grad, params=(),
         if len(y) != len(y_grad):
             raise ValueError(
                 '`y_grad` must have the same length of output values')
```

Comment:
Although it is out of the scope of this PR but we can also remove these backquotes, too.